### DISCUSSION TITLE
Pyrolysis Reaction

### DISCUSSION THREAD

**Discussion by AmirrezaHBSH** [firemodels/fds#12606] [Q&A]
I am studying a pyrolysis model involving three components present in the virgin biomass: cellulose, hemicellulose, and lignin. Each of these components undergoes four reactions denoted as R1 to R4. Reactions R1 to R3 are primary reactions involved in converting the virgin species into an active state, which subsequently undergo breakdown into lower molecular weight gases and char. The fourth reaction, R4, is a secondary reaction responsible for the breakdown of tar into lower molecular weight gases (Tar is gas).

Upon running this case, I've observed that the resulting (MPUA) versus temperature does not fix with previous data from identical samples. 

Do you know how I should proceed?

**Comment by mcgratta** [comment]
Could you be more specific. Are you comparing the FDS results to (1) an experiment, or (2) previous FDS results?

**Reply by AmirrezaHBSH** [reply]
I am validating it with previous numerical results that are not FDS.

**Comment by mcgratta** [comment]
I am not surprised. You will need to look at more results in addition to the final mass loss rate. Look at the evolution of the various material components. Check mass conservation, etc. Be aware that there is no "standard" set of reaction expressions and parameters that describe pyrolysis. This other model might be solving a slightly different set of reaction equations with a slightly different set of parameters.
